clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -1839,20 +1839,17 @@ private OffsetsForLeaderEpochResponse createLeaderEpochResponse() {
             .setPartitions(Arrays.asList(
                 new OffsetForLeaderPartitionResult()
                     .setPartition(0)
-                    .setErrorCode(Errors.NONE.code())
                     .setLeaderEpoch(1)
                     .setEndOffset(0),
                 new OffsetForLeaderPartitionResult()
                     .setPartition(1)
-                    .setErrorCode(Errors.NONE.code())
```

Comment:
In the case of the error code, I think it might be better to be explicit.

Reply:
I don't feel strong either ways so I will follow your suggestion.